src/Core/src/Handlers/Picker/PickerHandler.Android.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Specialized;
+using System.Linq;
+using Android.App;
+using Android.Text;
+using Android.Text.Style;
+using AResource = Android.Resource;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class PickerHandler : AbstractViewHandler<IPicker, NativePicker>
+	{
+		AlertDialog? _dialog;
+
+		protected override NativePicker CreateNativeView() =>
+			new NativePicker(Context);
+
+		protected override void ConnectHandler(NativePicker nativeView)
+		{
+			nativeView.FocusChange += OnFocusChange;
+			nativeView.Click += OnClick;
+
+			if (VirtualView != null && VirtualView.Items is INotifyCollectionChanged notifyCollection)
+				notifyCollection.CollectionChanged += OnCollectionChanged;
+
+			base.ConnectHandler(nativeView);
+		}
+
+		protected override void DisconnectHandler(NativePicker nativeView)
+		{
+			nativeView.FocusChange -= OnFocusChange;
+			nativeView.Click -= OnClick;
+
+			if (VirtualView != null && VirtualView.Items is INotifyCollectionChanged notifyCollection)
+				notifyCollection.CollectionChanged -= OnCollectionChanged;
```

Comment:
Are we unsubscribing when the Items/ItemsSource property changes? Like the whole list is assigned a new object?